test/CMakeLists.txt

```diff
@@ -6,3 +6,4 @@ koala_make_test(test_graph_recognition testGraphRecognition.cpp)
 koala_make_test(test_maximum_flow testMaximumFlow.cpp)
 koala_make_test(test_minimum_spanning_tree testMinimumSpanningTree.cpp)
 koala_make_test(test_dominating_set testDominatingSet.cpp)
+koala_make_test(test_cograph_alg testCographAlg.cpp)

```

Comment:
Please move the tests to their appropriate algorithms: independent set, coloring, vertex coloring, recognition